packages/serialize-request/lib/index.js

```diff
@@ -154,16 +155,28 @@ function serializeRequest(request, options = {}) {
  * Serialize request headers.
  *
  * @private
- * @param {Record<string, any>} headers
+ * @param {Headers | Record<string, any> | Iterable<[string, string]>} headers
```

Comment:
**issue:** line 11 of this file will also need to change to include the new header types, otherwise I get a Type error when I try to pass an iterable in. Updating the `BasicRequest` type to this works for me:

```js
/**
 * @typedef {object} BasicRequest
 * @property {Headers | Object<string, string> | Iterable<[string, string]>} [headers]
 *     The request HTTP headers.
 * @property {string} [method]
 *     The request HTTP method.
 * @property {string} [url]
 *     The request URL.
 */
```

Reply:
Just this one to go @Nazehs, everything else looking perfect. Sorry the line number is wrong – I can't add a suggestion for a line that wasn't edited in the PR but this needs to go near line 11